app/templates/views/guidance/index.html

```diff
@@ -41,7 +41,7 @@ <h2 class="padding-top-1" id="format-content">Format your content</h2>
   <h3>To create and format your message</h3>
   <ol class="list">
     <li>All messages start from a template</li>
-    <li>Click “<a href={{ url_for('.choose_template', service_id=current_service.id) }}>Send Messages</a>”. You’ll see existing templates.</li>
+    <li>Click “<a href={{ url_for('.choose_template', service_id=current_service.id) }}>Send Messages</a>”. You'll see existing templates.</li>
```

Comment:
Ah, the smart quotes persist here!  Let's get those fixed:

```suggestion
    <li>Click "<a href={{ url_for('.choose_template', service_id=current_service.id) }}>Send Messages</a>". You'll see existing templates.</li>
```